### PR TITLE
remove dependency on ramsey/uuid

### PR DESCRIPTION
**Pull request by othillo**
fixes https://github.com/broadway/broadway/issues/255
fixes https://github.com/broadway/broadway/issues/263

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -43,7 +42,8 @@
         "monolog/monolog": "~1.8",
         "symfony/console": "~2.4",
         "symfony/proxy-manager-bridge": "~2.4",
-        "phpunit/phpunit": "^4.8"
+        "phpunit/phpunit": "^4.8",
+        "ramsey/uuid": "^3.3"
```
Comment (wjzijderveld) [inline composer.json]: Why do we still need this in dev? It seems you removed all references already?

**Comment by wjzijderveld** [comment]
Can't we get rid of the `--ignore-platform-reqs` in `.travis.yml`?

**Comment by othillo** [comment]
I've removed the removal of composer.lock from this PR.

**Comment by othillo** [comment]
ramsey/uuid is still a dev dependency as the tests use the concrete binary uuid converter

**Pull request merged**